src/XAD/Complex.hpp

```diff
@@ -1481,20 +1492,20 @@ XAD_INLINE auto proj(const xad::FReal<T>& x) -> decltype(::xad::detail::proj_imp
 // different expr (derived1, derived2 - returns scalar)
 
 template <class T>
-XAD_INLINE complex<xad::AReal<T>> polar(const xad::AReal<T>& r, const xad::AReal<T>& theta)
+XAD_INLINE complex<xad::AReal<T>> polar(const xad::AReal<T>& r, const xad::AReal<T>& theta = 0)
 {
     return xad::detail::polar_impl(r, theta);
 }
 
 template <class T>
-XAD_INLINE complex<xad::FReal<T>> polar(const xad::FReal<T>& r, const xad::FReal<T>& theta)
+XAD_INLINE complex<xad::FReal<T>> polar(const xad::FReal<T>& r, const xad::FReal<T>& theta = 0)
 {
     return xad::detail::polar_impl(r, theta);
 }
 
 template <class Scalar, class Expr>
 XAD_INLINE complex<typename xad::ExprTraits<Expr>::value_type> polar(
-    const xad::Expression<Scalar, Expr>& r, const xad::Expression<Scalar, Expr>& theta)
+    const xad::Expression<Scalar, Expr>& r, const xad::Expression<Scalar, Expr>& theta = 0)
```

Comment:
you don't need the initializer for expressions - that won't work anyway. This overload is picked if an XAD expression is passed as second parameter, and the `=0` here is not needed in this case.